cypress/support/modularTransportClient/transportLayers/websocket.js

```diff
@@ -81,7 +81,25 @@ export default class Websocket extends AsyncTransportClient {
         res(event.data);
       };
```

Comment:
what happens if we get some error here ? wouldn't this cause issue in line 85 ? 
Can we add proper error handling here. For eg, if the msg is a string, this would throw an exception

Reply:
also what if msg does not have data field ? wouldnt that cause an exception